ui/src/app/pages/admin/user/admin-user-table.spec.tsx

```diff
@@ -28,6 +29,7 @@ describe('AdminUserTable', () => {
   beforeEach(() => {
     serverConfigStore.set({
       config: {
+        ...defaultServerConfig,
```

Comment:
I haven't investigated why this is only failing now but we should have been using the defaults in the first place.